nuxt-app/pages/form.vue

```diff
@@ -76,16 +77,16 @@
     </div>
 </template>
 
-<script setup>
+<script setup lang="ts">
 import { PieButton } from '@justeattakeaway/pie-button';
 import { PieSwitch } from '@justeattakeaway/pie-switch';
-
+import { PieInput } from '@justeattakeaway/pie-input';
 import { defineModel } from 'vue';
 
-const username = defineModel('username');
-const email = defineModel('email');
-const password = defineModel('password');
-const passwordConfirmation = defineModel('passwordConfirmation');
+const username = defineModel('username', { default: '' });
```

Comment:
For context, was added due to prevent 'undefined' being rendered in the username field until the page was hydrated? Pls correct me if I'm wrong @jamieomaguire!

Reply:
This was added because having discussed with potential component users, it is reasonable to expect a default value to be a string rather than undefined.

We did also notice the undefined thing as you mentioned, but I am not sure if that's a real issue to be concerned with

Reply:
We do the same thing in the NextJS app around line 10